worker/include/RTC/Codecs/PayloadDescriptorHandler.hpp

```diff
@@ -14,6 +16,65 @@ namespace RTC
 			virtual void Dump() const    = 0;
 		};
 
+		class PictureIdList
+		{
+			static constexpr uint16_t MaxCurrentLayerPictureIdNum{ 1000u };
+
+		public:
+			explicit PictureIdList()
+			{
+			}
+
+			~PictureIdList()
+			{
+				this->list.clear();
+			}
+
+			void Push(uint16_t pictureId, int16_t layer)
+			{
+				for (const auto& it : this->list)
+				{
+					// Layers can be changed only with ordered pictureId values.
+					// If pictureId is lower than the previous one, then it has rolled over the max value.
+					uint16_t diff = pictureId > it.first
+					                  ? pictureId - it.first
+					                  : pictureId + RTC::SeqManager<uint16_t, 15>::MaxValue - it.first;
+
+					if (diff > MaxCurrentLayerPictureIdNum)
+					{
+						this->list.pop_front();
+					}
+					else
+					{
+						break;
+					}
+				}
+
+				this->list.push_back({ pictureId, layer });
+			}
+
+			int16_t GetLayer(uint16_t pictureId) const
```

Comment:
What does this `GetLayer()` method return? An spatial layer? A temporal layer? Or what? If a spatial or temporal layer then let's please make it explicit in the method name.

Ok ok, I see that this is `PictureIdList` class that later is used as follows:
```c++
int16_t layer = this->spatialLayerPictureIdList.GetLayer(pictureId);
//
int16_t layer = this->temporalLayerPictureIdList.GetLayer(pictureId);
```

so it's ok.